invenio_administration/ext.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, app=None, entry_point_group="invenio_administration.views"):
         self.entry_point_group = entry_point_group
 
         self.administration = None
+        self.registered_extensions = []
```

Comment:
why is this needed inside the extension?

Reply:
leftover, thanks for spotting it